src/app-gocardless/app-gocardless.js

```diff
@@ -20,13 +20,7 @@ app.get('/link', function (req, res) {
 
 export { app as handlers };
 app.use(express.json());
-app.use(async (req, res, next) => {
-  let user = await validateUser(req, res);
-  if (!user) {
-    return;
-  }
-  next();
-});
+app.use(errorMiddleware);
```

Comment:
should this also use `validateUserMiddleware`?

Reply:
Oops, imported the wrong middleware